### ISSUE TITLE
How to get learning rate to decrease more slowly?

### ISSUE THREAD

**Issue by yajain** [CSAILVision/semantic-segmentation-pytorch#261]
How should I change the value of lr_pow in order to make my learning rate decrease more slowly. Currently the lr_pow is at a default value of 0.9.

**Comment by jayhsu0627** [comment]
By using `--lr_pow', default=0.9, type=float, help='power in poly to drop LR'`
